universal-application-tool-0.0.1/app/views/admin/AdminLayout.java

```diff
@@ -27,85 +16,38 @@ public AdminLayout(ViewUtils viewUtils) {
     super(viewUtils);
   }
 
-  private ContainerTag renderNavBar() {
-    String questionLink = controllers.admin.routes.QuestionController.index().url();
-    String programLink = controllers.admin.routes.AdminProgramController.index().url();
-    String versionLink = routes.AdminVersionController.index().url();
-    String intermediaryLink = routes.TrustedIntermediaryManagementController.index().url();
-    String logoutLink = org.pac4j.play.routes.LogoutController.logout().url();
-
-    ContainerTag headerIcon =
-        div(span("C"), span("F").withClasses(Styles.FONT_THIN))
-            .withClasses(BaseStyles.ADMIN_NAV_BAR);
-    ContainerTag headerTitle =
-        div()
-            .withClasses(
-                Styles.FONT_NORMAL, Styles.INLINE, Styles.PL_10, Styles.PY_0, Styles.TEXT_XL)
-            .with(span("Civi"), span("Form").withClasses(Styles.FONT_THIN));
-
-    ContainerTag adminHeader =
-        nav()
-            .with(headerIcon, headerTitle)
-            .with(headerLink("Questions", questionLink))
-            .with(headerLink("Programs", programLink))
-            .with(headerLink("Versions", versionLink))
-            .with(headerLink("Intermediaries", intermediaryLink))
-            .with(headerLink("Logout", logoutLink, Styles.FLOAT_RIGHT))
-            .withClasses(BaseStyles.NAV_STYLES);
-    return adminHeader;
-  }
-
-  public Content renderCentered(ContainerTag mainContent, String... mainStyles) {
-    String mainCenteredStyles =
-        StyleUtils.joinStyles(
-            Styles.PX_2, Styles.MAX_W_SCREEN_XL, Styles.MX_AUTO, StyleUtils.joinStyles(mainStyles));
-    return renderMain(mainContent, mainCenteredStyles);
-  }
-
-  public Content renderFull(ContainerTag mainDomContents, String... mainStyles) {
-    String mainFullStyles =
-        StyleUtils.joinStyles(Styles.FLEX, Styles.FLEX_ROW, StyleUtils.joinStyles(mainStyles));
-    return renderMain(mainDomContents, mainFullStyles);
-  }
-
-  private Content renderMain(ContainerTag mainContent, String... mainStyles) {
-    mainContent.withClasses(
-        Styles.BG_WHITE,
-        Styles.BORDER,
-        Styles.BORDER_GRAY_200,
-        Styles.MT_12,
-        Styles.OVERFLOW_Y_AUTO,
-        Styles.SHADOW_LG,
-        Styles.W_SCREEN,
-        StyleUtils.joinStyles(mainStyles));
-
-    return htmlContent(
-        head(tailwindStyles()),
-        body()
-            .with(renderNavBar())
-            .with(mainContent)
-            .with(viewUtils.makeLocalJsTag("main"))
-            .withClasses(
-                BaseStyles.BODY_GRADIENT_STYLE,
-                Styles.BOX_BORDER,
-                Styles.H_SCREEN,
-                Styles.W_SCREEN,
-                Styles.OVERFLOW_HIDDEN,
-                Styles.FLEX));
+  private final String BODY_STYLES =
```

Comment:
suggestion (non-blocking): Consider moving these to `AdminStyles.java` and removing the `_STYLES` suffix from them. That way you can reference them like `AdminStyles.BODY` (which will match the pattern I have going on in #1077). It also has the benefit of putting the definitions of these constants elsewhere, so that this class can focus on the other stuff.